.pre-commit-hooks.yaml

```diff
@@ -3,6 +3,7 @@
   description: Search and replace strings
   entry: search-and-replace
   language: ruby
+  language_version: 2.7.2
```

Comment:
is there a reason to use 2.7 over say like 3.3?

Reply:
That's what's used in https://github.com/PlasmaPy/PlasmaPy/pull/2455. Should we go with the more recent 3.3?

Reply:
Actually I guess there's an argument to be made for oldest compatibility.

Reply:
@cbachhuber  @mattlqx FYI, this does not work with pre-commit.ci since 2.7 is already EOL and pre-commit.ci does not support unmaintained language versions. My recommendation would be to update to 3.2 according to https://www.ruby-lang.org/en/downloads/branches/

Reply:
https://github.com/mattlqx/pre-commit-search-and-replace/pull/30